### PR TITLE
Add sensor data api

### PR DESCRIPTION
**Pull request by CunliangGeng**
To address issue #10

### CODE REVIEW

File: sensor_api/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+PerfectFit API for sensor data
```
Comment (svenvanderburg) [inline sensor_api/README.md:1]: Can you add some instructions on how to run the server?

File: sensor_api/perfectfit/run.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        __name__,
+        specification_dir='openapi/',
+        debug=True,
+        options={"swagger_url": ""}
+    )
+
+    options = {"swagger_ui": True}
+    app.add_api('openapi.yaml',
+                options=options,
+                arguments={'title': 'PerfectFit server'},
```
Comment (svenvanderburg) [inline sensor_api/perfectfit/run.py:16]:
```suggestion
                arguments={'title': 'PerfectFit sensor data server'},
```

File: sensor_api/perfectfit/api/sensor.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        sensor_type(str): "Accelerometer".
+
+    Returns:
+        HTTP responce: Responce to POST request.
+    """
+    req_data = request.data
+    req_data = req_data.decode('utf-8')
+    # TODO: valid userID
+    # valid, message = validator.validate(req_data, sensor_type)
+    valid = True
```
Comment (svenvanderburg) [inline sensor_api/perfectfit/api/sensor.py:16]: Add a TODO for writing data to database.

**Review by svenvanderburg — COMMENTED** [review]
Nice I really like this as a start! :)
You can do a few improvements, but really don't spend too much time on it, since it will change in the future anyway.

* I don't think this should be an installable package, just a script that runs the server makes more sense. Eventually we can wrap it in a docker image. It also reduces complexity as we can get rid of `setup.py`, `setup.cfg`, and `Makefile`.
* I think the name of the module `perfectfit` is confusing, as the whole project is called perfectfit. Maybe `app` makes more sense (I think that's generally used in flask).

File: sensor_api/README.md (hunk 1 above)
Comment (CunliangGeng) [reply sensor_api/README.md]: yep, it's added now.

File: sensor_api/perfectfit/api/sensor.py (hunk 3 above)
Comment (CunliangGeng) [reply sensor_api/perfectfit/api/sensor.py]: Actually it's there :-)

**Comment by CunliangGeng** [comment]
- `makefile` is used only for developing stage, so nice to keep it now and remove it when releasing code.
- `app` is used by flask, so I guess `perfectfitapp` would be better

**Comment by svenvanderburg** [comment]
> * `app` is already used by flask and would cause confuse if using it, so I guess `perfectfitapp` would be more specific and clear :-)

Then I would call it `sensor_api` or something like that, otherwise it still confuses with the overall project.

File: sensor_api/perfectfit/api/sensor.py (hunk 3 above)
Comment (svenvanderburg) [reply sensor_api/perfectfit/api/sensor.py]: Ah, I see! Sorry I need to look properly! 👓

**Comment by CunliangGeng** [comment]
> > * `app` is already used by flask and would cause confuse if using it, so I guess `perfectfitapp` would be more specific and clear :-)
> 
> Then I would call it `sensor_api` or something like that, otherwise it still confuses with the overall project.

OK, then let's call it `sensor_api` at this moment.

**Pull request merged**